junebug/workers.py

```diff
@@ -114,6 +128,13 @@ def _store_event(self, event):
 
     @inlineCallbacks
     def _forward_event(self, event):
+        '''Forward the event to the correct places.'''
+        yield self._forward_event_http(event)
+        if self.config.get('message_queue') is not None:
```

Comment:
Is there a reason this check is here rather than inside `_forward_event_amqp` like it is for `_forward_event_http`?